packages/cli/package-lock.json

```diff
@@ -17285,11 +17285,6 @@
         }
       }
     },
-    "google-libphonenumber": {
-      "version": "3.2.6",
-      "resolved": "https://registry.npmjs.org/google-libphonenumber/-/google-libphonenumber-3.2.6.tgz",
-      "integrity": "sha512-6QCQAaKJlSd/1dUqvdQf7zzfb3uiZHsG8yhCfOdCVRfMuPZ/VDIEB47y5SYwjPQJPs7ebfW5jj6PeobB9JJ4JA=="
-    },
```

Comment:
Surprised this gets deleted too here.
@gastonponti can you confirm?

Reply:
mmm, yes it's weird. But technically we are using the `yarn.lock`, not the `package-lock.json`
And if you check the first lines its says:
```
  "name": "@celo/celocli",
  "version": "0.0.27",
```
I assume that was an `npm install` that someone commited some time ago
I strongly believe that this won't generate any problem. I would rule it out